### PR TITLE
[OSDEV-728] API. Sector values are missing from GET /facilities response. 

### PR DESCRIPTION
**Pull request by roman-stolar**
[OSDEV-728](https://opensupplyhub.atlassian.net/browse/OSDEV-728) API. Sector values are missing from GET /facilities response.

- Added 'sector' data response to endpoint **api/facilities/** similar as we got in **api/facilities/{id}**.
- Excluded _'sector'_ field if parameter _'detail'_ not set or equal _'true'_
- Updated docs for Swagger.

### CODE REVIEW

File: src/django/api/serializers/facility/facility_index_details_serializer.py [hunk 1]
```diff
@@ -344,78 +338,3 @@ def get_created_from(self, facility):
 
     def get_has_inexact_coordinates(self, facility):
         return False
-
-    def get_sector(self, facility):
-        user_can_see_detail = can_user_see_detail(self)
-
-        use_main_created_at = is_created_at_main_date(self)
```
Comment (killev) [inline src/django/api/serializers/facility/facility_index_details_serializer.py:351]: Why did you remove this line, it used there (Roma this is example please ignore)

Comment (roman-stolar) [reply src/django/api/serializers/facility/facility_index_details_serializer.py:351]: @killev As we for now we use **sector** data both in **/facilities/** & **/facilities/<id>** endpoints and _facilities_index_details_serializer.py_ inherit from _facilities_index_serializer.py_ we don't need duplicate **getSector** method.

Comment (killev) [reply src/django/api/serializers/facility/facility_index_details_serializer.py:351]: @roman-stolar sorry, I demonstrated functionality of GitHub, Please ignore my comment above :)

File: src/django/api/serializers/facility/utils.py [hunk 2]
```diff
@@ -450,3 +450,57 @@ def depromote_unspecified_items(items: list):
         k['values'][0] == 'Unspecified' and
         len(k['values']) == 1),
         reverse=False)
+
+
+def format_sectors(items,
```
Comment (vladsha-dev) [inline src/django/api/serializers/facility/utils.py:455]: Nice idea to move this functionality to a separate function. I think we could apply more refactoring techniques to this function.

```
def format_sectors(items, claims, date_field_to_sort, use_main_created_at, user_can_see_detail):
    def is_contributor_visible(item, is_claim):
        if is_claim:
                 return user_can_see_detail
        return (user_can_see_detail and item['source']['is_active']
                and item['source']['is_public'] and item['has_active_complete_match'])

    def format_sector_data(entity, is_claim):
        created_at = format_date(entity['created_at']) if use_main_created_at else None
        return {
            'created_at': created_at,
            'updated_at': format_date(entity['updated_at']),
            'contributor_id': get_contributor_id_from_facilityindexnew(entity['contributor'],
                                                                       is_contributor_visible(entity, is_claim)),
            'contributor_name': get_contributor_name_from_facilityindexnew(entity['contributor'],
                                                                           is_contributor_visible(entity, is_claim)),
            'values': entity['sector'],
            'is_from_claim': is_claim
        }

    item_sectors = [format_sector_data(item, False) for item in items]
    claim_sectors = [format_sector_data(claim, True) for claim in claims]

    sorted_item_sectors = sorted(item_sectors, key=lambda i: i[date_field_to_sort], reverse=True)

    return claim_sectors + depromote_unspecified_items(sorted_item_sectors)
```

**Review by vladsha-dev — CHANGES_REQUESTED** [review]
Please, add a note to the RELEASE-NOTES.md file.

**Comment by vladsha-dev** [comment]
> Please, add a note to the RELEASE-NOTES.md file.

Done

**Review by vladsha-dev — APPROVED** [review]
LGTM
Tested locally.

**Review by killev — APPROVED** [review]
Looks good. Approved

**Comment by VadimKovalenkoSNF** [comment]
@roman-stolar I was unable to see 'sector' property in the response. Could you share your curl request to the localhost? I used this one:
`curl --location 'http://localhost:6543/api/facilities/' \
--header 'Authorization: Token <My_API_Token>'`

I applied migrations using `scripts/update`. Did I miss something?

**Comment by roman-stolar** [comment]
> @roman-stolar I was unable to see 'sector' property in the response. Could you share your curl request to the localhost? I used this one: `curl --location 'http://localhost:6543/api/facilities/' \ --header 'Authorization: Token <My_API_Token>'`
> 
> I applied migrations using `scripts/update`. Did I miss something?

@VadimKovalenkoSNF You should use 'http://localhost:6543/api/facilities/?detail=true'
